Add config to warn public deployment exposure in UI (#18557)
mistercrunch/Airflow: docs/apache-airflow/security/webserver.rst

    [webserver]
    x_frame_enabled = False

> Disable Deployment Exposure Warning
> ------------------------------------------------------

Reviewer: ```suggestion
Disable Deployment Exposure Warning
---------------------------------------
```

> Airflow warns when recent requests are made to /robot.txt. To disable this warning set the below:

Reviewer: ```suggestion
Airflow warns when recent requests are made to `/robot.txt`. To disable this warning set ``warn_deployment_exposure`` to ``False`` as below:
```

    [webserver]
    x_frame_enabled = False

Disable Deployment Exposure Warning
---------------------------------------

> Airflow warns when recent requests are made to `/robot.txt`. To disable this warning set ``warn_deployment_exposure`` to

Reviewer: ```suggestion
Airflow warns when recent requests are made to ``/robot.txt``. To disable this warning set ``warn_deployment_exposure`` to
```
